bridge/discord/discord.go

```diff
@@ -423,7 +424,7 @@ func (b *Bdiscord) replaceUserMentions(text string) string {
 		if err != nil {
 			return m
 		}
-		return member.User.Mention()
+		return member.User.Mention() + " " + m[lastSpace+2:]
```

Comment:
Change this to `return strings.Replace(m, "@"+mention, member.User.Mention(), -1)`  this will fix the issue I talked about in my previous comment.